packages/v4/src/content/accessibility/skip-to-content/skip-to-content.md

```diff
@@ -0,0 +1,16 @@
+---
+id: Skip to content
+section: components
+---
+
+A **skip to content** is a component that allows screen reader and keyboard users to bypass navigation rather than tabbing through it.
+
+**Keyboard users** should be able to find the skip to content using **Tab** at the start of a page with navigation. They should be able to select the skip to content using **Enter**.
+
+**Screen reader users** should also be able to navigate to skip to content using **Tab** and select it to skip past the navigation of a page.
+
+A skip to content does not trap tab focus. It is the first tabbable element on a page, but can be tabbed past and into the navigation of a page if the user prefers.
+
+## To make skip to content accessible:
+- Make sure the skip to content can be accessed with **Tab** 
+- Give the skip to content a tabindex of -1 so that it can receive programmatic focus for keyboard and screen reader users. For more info on this tabindex, see [the WebAIM page for tabindex](https://webaim.org/techniques/keyboard/tabindex#zero-negative-one).
```

Comment:
I may be completely backwards in my understanding, but if I give it the skip to content a tabindex of -1, wouldn't that mean that I could NOT tab to it anymore? Isn't that the opposite of what we want?

Reply:
You're right! The target of the skip link should have tabindex -1 to make it work consistently on IE and Chrome, but the element itself needs to be focusable.